realm/src/androidTest/java/io/realm/RealmInMemoryTest.java

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
I just went through the RealmTest, most of those cases should not be affected by the durability at all. So I choose to Have those a few special cases here for now. And also did checking with the relevent IOS cases, in addition to those basic cases we have here, some other performance cases are also created. Maybe we should have same performance cases as well to avoid performance regression?